interfaces/cython/cantera/ck2yaml.py

```diff
@@ -2093,16 +2064,36 @@ def convert_mech(input_file, thermo_file=None, transport_file=None,
         surface_names = parser.write_yaml(name=phase_name, out_name=out_name)
         if not quiet:
             nReactions = len(parser.reactions) + sum(len(surf.reactions) for surf in parser.surfaces)
-            print('Wrote YAML mechanism file to {0!r}.'.format(out_name))
-            print('Mechanism contains {0} species and {1} reactions.'.format(len(parser.species_list), nReactions))
-        return surface_names
+            logger.info('Wrote YAML mechanism file to {0!r}.'.format(out_name))
+            logger.info('Mechanism contains {0} species and {1} reactions.'.format(
+                        len(parser.species_list), nReactions))
+        return parser, surface_names
+
+    def show_duplicate_reactions(self, error_message):
+        reactions = []
+        for line in error_message.split('\n'):
+            match = re.match('>.*# Reaction ([0-9]+)', line)
+            if match:
+                reactions.append(int(match.group(1))-1)
+
+        if len(reactions) != 2:
+            logger.warning(error_message)
+            return
+
+        equation = str(self.reactions[0])
```

Comment:
This looks a little weird to me, and it doesn't appear to print the correct reaction if I reorder the reactions. For instance, this reactions section:
```
REACTIONS
R1A => R1B                      1.0e-10 0.0 1000.0
R1A+R1B+M <=> P1+H+M            1.0e-2    0.0     5000.0
R1A+R1B <=> P1+H                1.0e19    0.0     5000.0
P1+H => R1A+R1B                       1.0e19    0.0     5000.0
END
```
gives me this error message:
```
Validating mechanism...
FAILED
Undeclared duplicate reaction R1A => R1B
found on lines 13 and 14 of the kinetics input file.
```
where lines 13 and 14 refer to the last two reactions, but `R1A => R1B` is clearly not one of the duplicates, but it is the first reaction in the list. Should this be
```
self.reactions[reactions[0]]
```
?

Reply:
Yes, indeed. 😂